source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -15,7 +16,19 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 5.0
```

Comment:
Question: Did you mean to add 5.0 as a Java version? I thought 4.3 was the next version being released.